ts/table-grid/table-grid.controller.ts

```diff
@@ -150,6 +151,10 @@ module Lui.Directives {
 				return tmp.textContent || tmp.innerText || "";
 			};
 
+			$scope.updateCheckboxes = () => {
+				_.each($scope.datas, (data: any) => { data.isChecked = $scope.allChecked.value; });
+			};
```

Comment:
It is a good idea but something is missing. I have to add a method on each checkbox to unselect master checkbox if necessary.